AirportTicketBookingSystem.Tests/DomainTests/ServicesTests/FlightServiceTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Moq;
+using FluentAssertions;
+using AutoFixture;
+using AirportTicketBookingSystem.Domain.Models;
+using AirportTicketBookingSystem.Domain.Repositories;
+using AirportTicketBookingSystem.Domain.Services;
+public class FlightServiceTests
+{
+    private readonly Mock<IFlightRepository> _mockFlightRepo;
+    private readonly FlightService _flightService;
+    private readonly Fixture _fixture;
+
+    public FlightServiceTests()
+    {
+        _mockFlightRepo = new Mock<IFlightRepository>();
+        _flightService = new FlightService(_mockFlightRepo.Object);
+        _fixture = new Fixture();
+    }
+
+    [Fact]
+    public void AddFlight_ShouldCallRepositoryAddFlight()
```

Comment:
A better name to stick to the convention: `AddFlight_ShouldSucceed_ByDefault`. Lets do this change with the other similar naming.